test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/restApi.test.js

```diff
@@ -157,4 +161,23 @@ describe('#compileRestApi()', () => {
     awsCompileApigEvents.serverless.service.provider.vpcEndpointIds = ['id1'];
     expect(() => awsCompileApigEvents.compileRestApi()).to.throw(Error);
   });
+
+  it('should not disable default execute-api endpoint', () => {
```

Comment:
All newly configured tests should use `runServerless` instead of the "old" way of tests such as here. For details please see https://github.com/serverless/serverless/blob/master/test/README.md

For an example, you can refer to validators test: https://github.com/serverless/serverless/blob/80511a4b17e77e22cf8b20d1ce50eef7506d4f7f/test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/requestValidator.test.js#L9